src/lib/StdBuf.h

```diff
@@ -46,7 +46,7 @@ class StdBuf
 	// Will take over buffer ownership. Copies data if specified.
 	// Note: Construct with Buf2.getRef() to construct a reference (This will work for a constant Buf2, too)
 	StdBuf(StdBuf & Buf2, bool fCopy = false)
-			: fRef(true), pData(nullptr), iSize(0)
+			: pData(nullptr)
```

Comment:
Are all these explicit `pData` initializations required? We're initializing in line 94.